### PR TITLE
Hotfix/suit 11910 windows ipc fix

### PR DESCRIPTION
**Pull request by givehug**
Fix closing ipc connection in child process, error messages on windows.

### CODE REVIEW

File: lib/testLauncher/ipc/server.js [hunk 1]
```diff
@@ -27,24 +28,27 @@ const handleMessageFromClient = handleMessage(state);
...
-		socket.on('data', handleMessageFromClient);
+		socket.on('error', err => logger.debug('IPC server socket error', err.errno));
 		socket.on('close', () => delete state.connections[socket.id]);
+		socket.on('data', handleMessageFromClient);
 		socket.unref();
 		write(socket, messageId.SETUP_SESSION, {config});
 	});
 
 	server.on('error', err => {
 		if (err.errno === 'EADDRINUSE') {
```
Comment (AntonSBilan) [inline lib/testLauncher/ipc/server.js]: When we can have this message? Does it makes sense for user to know about it as there is no way user can specified port or know in advance what port will be used by launcher

Comment (givehug) [reply lib/testLauncher/ipc/server.js]: I really don't know how to reproduce this error. IPC server should be started on any available port.

Comment (AntonSBilan) [reply lib/testLauncher/ipc/server.js]: So maybe we can have a message that all ports are busy or node cannot find available port?

Comment (givehug) [reply lib/testLauncher/ipc/server.js]: I don't know how to reproduce any of these scenarios, so I can't say which message would be better. We can try to investigate.

**Pull request merged**